website/public/main.css

```
@@ -718,6 +718,59 @@ kbd.keystroke {
     bottom: 0;
   }
 }
+.show-in-mobile {
+  display: none;
+}
+#supported-editors {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  row-gap: 2em;
+  text-align: center;
+  padding: 2em;
+}
+#supported-editors .logo {
+  width: 2rem;
+  height: 2rem;
+}
+#supported-editors a {
+  background-color: rgb(240, 219, 79);
+  font-weight: bold;
+  border-radius: 0.5rem;
+  margin: 0 0.5rem 0 0.5rem;
+  padding-bottom: 0.5rem;
+  padding-top: 0.5rem;
+  text-decoration: none;
+}
+#supported-editors a {
+  color: rgb(50, 51, 48);
+}
+@media only screen and (max-width: 43em) {
+  /* Hide the feature, JShint, Deno columns and stable, easy npm install rows*/
+  .hide-in-mobile {
+    display: none;
+  }
+
+  /* show elements that are only for mobile */
+  .show-in-mobile {
+    display: initial;
+  }
```

Nitpick: These comments seem unnecessary.
```suggestion
  .hide-in-mobile {
    display: none;
  }
  .show-in-mobile {
    display: initial;
  }
```